src/posts/action.md

```diff
@@ -3,56 +3,56 @@ title: Take action
-- **Protest**: join [one of the protests](/protests) or [organize one yourself](/organizing-a-protest).
-- **Sign petitions**: [International AI Treaty](https://aitreaty.org), [Ban Superintelligence](https://chng.it/Djjfj2Gmpk), [Demand responsible AI](https://www.change.org/p/artificial-intelligence-time-is-running-out-for-responsible-ai-development-91f0a02c-130a-46e1-9e55-70d6b274f4df), or one of the **national petitions**: [UK](https://petition.parliament.uk/petitions/639956), [AUS](https://www.aph.gov.au/e-petitions/petition/EN5163), [NL](https://aipetitie.nl)
+- **Join** the [Discord](https://discord.gg/2XXWXvErfA), where most of the collaboration happens.
+- **Find** or create a [local PauseAI community](/communities).
+- **Participate** in a protest, either on your own or with your local group. Find these on [event](/events).
+- **Discuss** how you can best contribute: fill out [this form](https://airtable.com/embed/appWPTGqZmUcs3NWu/pagoxRuCai4OYJEHt/form).
+- [**Donate**](/donate) to PauseAI or buy some merchandise in our [store](https://pauseai-shop.fourthwall.com/). The revenue goes directly toward PauseAI's initiatives to reduce AI risks.
+- **Follow** our [social media channels](https://linktr.ee/pauseai) and stay updated. Your local PauseAI region may also have social media pages.
 
-## Join the movement
 
-- Join or create a [local community](/communities) or [event](/events).
-- Join the PauseAI [Discord](https://discord.gg/2XXWXvErfA), where most of the collaboration happens.
-- Check out the [vacancies](/vacancies).
-- If you want to volunteer, fill out [this form](https://airtable.com/embed/appWPTGqZmUcs3NWu/pagoxRuCai4OYJEHt/form).
-- [**Donate**](/donate) to PauseAI or buy some merchandise in our [store](https://pauseai-shop.fourthwall.com/).
-- **Follow** our [social media channels](https://linktr.ee/pauseai) and stay updated,
-  <!-- ICONS in a line like the footer would be better -->
+## For specific people
 
-## If you...
+### If you work in AI
+
+- **DON'T WORK TOWARD STRONGER OR FASTER AI!** If you have some cool idea on how we can make AI systems 10x faster, please don't build it / spread it / talk about it. We need to slow down AI development, not speed it up.
+- **Talk to your management and colleagues** about the risks. Get them to take an institutional position toward mitigating risk over profit. Encourage implementation of standard risk mitigation procedures and anonymous reporting.
+- **Hold a seminar** on AI safety at your workplace. Check out these [slides](https://drive.google.com/drive/u/1/folders/1p9VtopzMV6Xpk4p6EGYUTna4fLE6G8hd) and [talks and videos](https://www.youtube.com/playlist?list=PLI46NoubGtIJa0JVCBR-9CayxCOmU0EJt) for inspiration.
+- **Sign** the [Statement on AI Risk](https://www.safe.ai/statement-on-ai-risk).
+- **Reach out** by email or [on Signal](https://signal.org/) if you have anything critical to share - username: pauseainyc.11
```

Comment:
Let's link to Joep's mail here (unless we have a better one).
Also, I'd put "(@pauseainyc.11)" or something alike after the Signal link so it's cleaner and clearer that it's related to that app.